benchmarks/python/test_embedding.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
```

Comment:
@jjsjann123 where is the validation code which raised the tolerance issue?

Reply:
Ahh nvm, I see that you are unable to confirm if embedding benchmarks passed since there were other benchmarks with tolerance issues

Reply:
Yeah CI failures seem to be just noisy and not from this test.

My local failure are the good old assert on no events founds. :cry:

Reply:
Yeah I see that on my local workstation as well. Strangely, the profile is CPU only. The event that is otherwise recorded as CUDA event is reported as a CPU event :/

Reply:
Yeah I see that on my local workstation as well. Strangely, the profile is CPU only. The event that is otherwise recorded as CUDA event is reported as a CPU event :/